doc/rtd/reference/cli.rst

```diff
@@ -91,7 +91,10 @@ re-run all stages as it did on first boot.
   cleans config files for ssh daemon. Argument `network` removes all generated
   config files for network. Argument `datasource` removes files and/or configs
   written by current datasource. `all` removes config files of all types.
-
+* :command:`--seed`: Remove the cloud-init seed directory (e.g., :file:`/var/lib/cloud/seed/`) 
+  which stores instance metadata used for simulating a cloud environment. 
```

Comment:
```suggestion
  which stores instance metadata used initializing a datasource. 
```

`simulating` isn't really the right word here since the seed directory is a valid and expected configuration source for many datasources.